libs/wire-api/src/Wire/API/Routes/Internal/Galley.hs

```diff
@@ -56,128 +56,40 @@ import Wire.API.Team.Member
 import Wire.API.Team.SearchVisibility
 import Wire.API.User.Client
 
-type LegalHoldFeatureStatusChangeErrors =
```

Comment:
👏

Reply:
I did this fix at some point already, but it got discarded.